### ISSUE TITLE
Failed to fetch a lazy token

### ISSUE THREAD

**Issue by AxelWine** [realcoloride/node_characterai#20]
When you try to login with a guest (or even with a real token) the API returns an error, which makes it impossible to use the chats, I don't know why exactly but I've been testing it for several days in different environments and it always gives me the same error, while before it did work normally.

**Testing code:**
```
const CharacterAI = require('node_characterai');
const characterAI = new CharacterAI();
 
(async() => {
    await characterAI.authenticateAsGuest();
 
    const characterId = "8_1NyR8w1dOXmI1uWaieQcd147hecbdIK7CeEAIrdJw" // Discord moderator
 
    const chat = await characterAI.createOrContinueChat(characterId);
    const response = await chat.sendAndAwaitResponse('Hello discord mod!', true)
 
    console.log(response);
})();
```
**Error:**

![imagen](https://user-images.githubusercontent.com/82121874/227801007-77696d29-7520-4964-84bc-0159097fd1ed.png)

**Comment by tylerstraub** [comment]
I haven't found a solution yet but I can confirm that it is happening on my end now as well.

**Comment by realcoloride** [comment]
I'm aware and i'm trying to fix it #17

**Comment by realcoloride** [comment]
Fixed in https://github.com/realcoloride/node_characterai/issues/17
